cmd/oras/internal/errors/errors_test.go

```diff
@@ -53,3 +53,36 @@ func TestCheckMutuallyExclusiveFlags(t *testing.T) {
 		})
 	}
 }
+
+func TestCheckRequiredTogetherFlags(t *testing.T) {
+	fs := &pflag.FlagSet{}
```

Comment:
Should add a case to check the case when none required flags are used. E.g. `--hello` and `--world` are required to use together but none of it is changed, `CheckRequiredTogetherFlags` should not return error.

Reply:
added a test